CNAME

```diff
@@ -1 +0,0 @@
-wpaustralia.org
```

Comment:
This file needs to remain until we have a host hosting setup for https://wpaustralia.org